src/gsi/m_berror_stats_reg.f90

```diff
@@ -830,6 +831,10 @@ subroutine berror_read_wgt_reg(msig,mlat,corz,corp,hwll,hwllp,vz,rlsig,varq,qopt
         do i=0,mlat+1
            hwllp(i,n)=hwll(i,1,nrf3_q)
         end do
+        if ( l_rtma3d ) then  ! tuning stddev and length for analysis of gust in 3drtma test run
```

Comment:
If the users forget to set values (corp_gust and/or hwllp_gust),  the code will use the above (default) q values without any warning message and the user might be unable to notice this .  Would it be better to let GSI abort with explanation if negative values are encountered here?

Reply:
@TingLei-NOAA
Hi Ting,

The purpose for introducing corp_gust and hwllp_gust is to give the user the options to tune the background error (co)variances. The default values  have been tuned to be optimal for general cases and good enough to be used in most cases, and user does not need to concern about it. When corp_gust or hwllp_gust is negative, it means to use the default values for analysis of wind gust, so GSI does not need to abort.  

I will add more clear explanation and a printout to remind the user of the used values of the parameters of background and observations error (co)variances for wind gust in GSI code. Thank you for pointing it out!

-Gang

Reply:
@GangZhao-NOAA  But on line 832, it is the hwll of q on the first level being used as the default for wind gust, which could be tuned for considerations on q itself.  So, is such default reliable ?